feature-libs/asm/root/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './facade/index';
+export * from './model/index';
```

Comment:
We can put all exported stuff in index.ts
and `export * from './index';` in public_api.ts.